src/App.js

```diff
@@ -1,29 +1,13 @@
-import logo from './logo.svg';
-import './App.css';
+import React from 'react';
+import Calculator from './components/calculator';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          {' '}
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+// eslint-disable-next-line react/prefer-stateless-function
+class App extends React.Component {
+  render() {
+    return (
+      <Calculator />
+    );
+  }
 }
```

Comment:
# Thanks for the review @mikethreels.

I want to say that I made all the required changes but the only question that I have is this one:
`class App` is a stateless component so I disabled the eslint error because it says it needs to be a function. If this is not allowed how can I solved with out doing this?

Reply:
BTW @mikethreels I made a mistake and sent you my PR's link as feedback 🤣 🤣 🤣 🤣 🤣

Reply:
In this case, adding a constructor to your class-based component should help you fix the issue.

```javascript
constructor() {
    super();
    this.state = {};
  }
```